runtime/src/accounts_db.rs

```diff
@@ -1573,7 +1624,7 @@ impl AccountsDB {
                                     }
                                 }
 
-                                Some((**pubkey, *account.hash))
+                                Some((**pubkey, *account.hash, balance))
```

Comment:
Well, I wanted AtomicU128, but it looks like this still isn't stable: https://github.com/rust-lang/rust/issues/32976

Reply:
done https://github.com/solana-labs/solana/pull/11927/commits/51d532f0b1f8e871f1ef3cf7bcf159701665515a